src/storer.ts

```diff
@@ -0,0 +1,45 @@
+import {WebPage} from "./web-page";
+/**
+ * Created by lukas on 3/27/17.
+ */
+export class Storer {
+
+    static azure = require('azure');
+    static crypto = require('crypto');
+
+    static blobService = Storer.azure.createBlobService('wetstorage',
+        'x7P3BHTnMpY+JB2dAHnzkaHVoz080IUSthfYDhTPFtYWNzIuwn70szGE+vIHn5S4BPad6gioTdlLafxkpszGHQ==');
```

Comment:
I'm sorry, this really shouldn't be here. I'll fix it when I get home. For now I regenerated the keys to make this one invalid.